examples/00-load/create-unstructured-surface.py

```diff
@@ -196,8 +196,21 @@
     ]
 )
 
-# 10 is just vtk.VTK_TETRA
-celltypes = np.array([10, 10, 10, 10, 10, 10, 10, 10, 10, 10], dtype=np.uint8)
+celltypes = np.array(
+    [
+        CellType.TETRA,
+        CellType.TETRA,
+        CellType.TETRA,
+        CellType.TETRA,
+        CellType.TETRA,
+        CellType.TETRA,
+        CellType.TETRA,
+        CellType.TETRA,
+        CellType.TETRA,
+        CellType.TETRA,
+    ],
+    dtype=np.uint8,
+)
```

Comment:
I think this is a bit excessive. I'd just use either `[CellType.TETRA] * 10`, or `np.full(10, fill_value=CellType.TETRA, dtype=np.uint8)`. The latter is idiomatic NumPy.

I also have half a mind to remove all these magic numbers (like the 10 in `np.full(10, ...)`), but it's probably not worth the effort for an example.

Reply:
```suggestion
celltypes = np.full(10, fill_value=CellType.TETRA, dtype=np.uint8)
```